sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/compatibility/AzureSpringBootVersionVerifier.java

```diff
@@ -113,9 +119,10 @@ private boolean springBootVersionMatches() {
 
     private boolean matchSpringBootVersionFromManifest(String acceptedVersion) throws FileNotFoundException {
         String version = this.getVersionFromManifest();
-        LOGGER.debug("Version found in Boot manifest [{}]", version);
+        LOGGER.debug("Currently running on Spring Boot version [{}], trying to match it in Spring Cloud Azure "
```

Comment:
`Currently running on Spring Boot version [{}], trying to match it with Spring Cloud Azure accepted version [{}].`